ravenframework/GridEntities.py

```diff
@@ -504,9 +504,9 @@ def initialize(self, initDictionary=None):
       if self.gridContainer['transformationMethods'] is not None:
         if varName in self.gridContainer['transformationMethods']:
           self.gridContainer['gridVectors'][varName] = np.asarray([self.gridContainer['transformationMethods'][varName][0](coor) for coor in self.gridContainer['gridVectors'][varName]])
-      pointByVar[varId]                              = np.int(np.shape(self.gridContainer['gridVectors'][varName])[0])
+      pointByVar[varId]                              = int(np.shape(self.gridContainer['gridVectors'][varName])[0])
     self.gridContainer['gridShape']                  = tuple   (pointByVar)                             # tuple of the grid shape
-    self.gridContainer['gridLength']                 = np.int(np.prod (np.asarray(pointByVar, dtype=np.float64))) # total number of point on the grid
+    self.gridContainer['gridLength']                 = int(np.prod (np.asarray(pointByVar, dtype=np.float64))) # total number of point on the grid
```

Comment:
warning deprecation of ```np.int```

Reply:
Just as a warning, I pulled this change into another branch and it has been merged #2057